src/applications/check-in/pre-check-in/components/Footer.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
 
-const Footer = () => (
+const Footer = ({ additionalInformation = <> </> }) => (
```

Comment:
This is super minor and fine as is, but I would probably remove the default param here and make it an optional parameter in propTypes. The reason is that this may cause a new react fragment to render created on each render. 

https://github.com/facebook/react/issues/16905

Reply:
oh yeah, I knew something was off about that. Thanks Dan!

Reply:
Oh yeah I knew something seemed off with that. Thanks Dan!